### DISCUSSION TITLE
Using the geo filter

### DISCUSSION THREAD

**Discussion by jean-louis67** [photoprism/photoprism#3477] [Q&A]
Hello
I am trying to identify my recent photos without GPS coordinates.

1. I select the photos from June 2023. The pop-up window tells me 226 pĥotos found.
2. I add the  filter _geo:yes_ The pop-up window tells me 179 pĥotos found. It's entirely plausible. But as it is precisely the other photos that interest me:
3. I change the filter: _geo:no_
The pop-up window tells me 226 pĥotos found… whereas logically only 48 photos should be found.

Did I misunderstand the filter?

**Comment by graciousgrey** [comment]
Most of our boolean search filters only work with the "yes" condition. However, with today's changes, geo:false will work in our development preview :)

See also:
- https://github.com/photoprism/photoprism/issues/3493

**Reply by jean-louis67** [reply]
Great, thanks a lot!
Generally speaking, it seems to me that it is not intuitive that a boolean test att:yes/true will not work with att:no/false. And it is difficult to predict what will really interest this or that user. Does it make sense to imagine a _not_ operator that could apply like this: _not (geo:true)_? This would avoid having to develop all the cases, attribute by attribute. It's not a priority for me, of course.

**Reply by graciousgrey** [reply]
We do already have an open issue for a not operator. This will be very useful also for non boolean search filters :)
